src/aind_ephys_rig_qc/qc_figures.py

```diff
@@ -56,16 +72,142 @@ def plot_power_spectrum(data, sample_rate, stream_name):
     """
 
     fig = Figure(figsize=(10, 4))
-    ax = fig.subplots()
+    ax = fig.subplots(2, 1, gridspec_kw={"height_ratios": [2, 1]})
```

Comment:
```suggestion
    ax = fig.subplots(2, 1, gridspec_kw={"height_ratios": [2, 1]}, sharex=True)
```
So that the two plots are aligned